# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

GoToPosActionGoal action_goal
GoToPosActionResult action_result
GoToPosActionFeedback action_feedback
